src/index.js

```diff
@@ -47,12 +47,13 @@ function handleExtendsNodes(tree, options, messages) {
         locals = JSON.parse(extendsNode.attrs.locals);
       } catch {}
     }
-    options.expressions.locals = merge(options.expressions.locals, locals);
-    options.plugins.push(expressions(options.expressions));
+
+    var options_expressions = merge(options.expressions, {locals});
+    var plugins = [...options.plugins, expressions(options_expressions)];
```

Comment:
1. var ?

Reply:
😊 sorry, I've no experience in JS, I've educaded myself a little more on scoping. fixed.